DEVELOPMENT.md

```diff
@@ -34,3 +34,8 @@ Then whenever a file is edited, it will be re-compiled and available for reloadi
 If you are debugging a customer site, you can type `addressfinderDebugMode()` into the javascript console. This will reinitialise the widget,
 with the debug flag set to true, so you can see console logs from the addressfinder-webpage-tools npm package.
 This works in Chrome and FireFox
+
+# Trouble Shooting
```

Comment:
This is generally one word I think. Otherwise it refers to a problem caused by a faulty firearm 😆